packages/components/select/src/select.vue

```diff
@@ -554,8 +557,15 @@ export default defineComponent({
       ),
     ])
 
+    // if in form and use statusIcon, the width of the icon needs to be subtracted, fix #13526
     const selectTagsStyle = computed(() => ({
-      maxWidth: `${unref(inputWidth) - 32}px`,
+      maxWidth: `${
+        unref(inputWidth) -
+        32 -
+        (needStatusIcon.value && validateState.value && validateIcon.value
+          ? 22
```

Comment:
## Question

22

Is it the resulting value for 14 + 8 like in the picture?

<img width="1231" alt="capture" src="https://github.com/element-plus/element-plus/assets/27342882/393ab02a-79bd-452d-93df-05a95d3ce883">

Reply:
yes